app/src/main/res/layout/activity_main.xml

```diff
@@ -7,24 +7,34 @@
     tools:context=".MainActivity">
 
     <Button
-        android:id="@+id/kotlinButton"
+        android:id="@+id/dateTime"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginTop="64dp"
-        android:text="Kotlin Click"
+        android:text="date Time"
```

Comment:
Let's name it "Date time picker"
P.s. we can hardcode here since it is code example